nanoFramework.Json/JsonConvert.cs

```diff
@@ -59,10 +57,63 @@ public static string SerializeObject(object oSource)
         /// <returns></returns>
         public static object DeserializeObject(string sourceString, Type type)
         {
+            if (type == typeof(string))
+            {
+                return DeserializeStringObject(sourceString);
+            }
+
             var dserResult = Deserialize(sourceString);
             return PopulateObject((JsonToken)dserResult, type, "/");
         }
 
+        private static char GetEscapableCharKeyBasedOnValue(char inputChar)
+        {
+            foreach (var item in JsonSerializer.EscapableCharactersMapping.Keys)
+            {
+                var value = (char)JsonSerializer.EscapableCharactersMapping[item];
+                if (value == inputChar)
+                {
+                    return (char)item;
+                }
+            }
+
+            throw new InvalidOperationException($"Value {inputChar} is not supported.");
```

Comment:
We're avoiding messages in exceptions to save flash space and decrease assembly size. Better leave this one empty too.
In case anyone looks at the code it will become "obvious" what the problem is.

```suggestion
            // in case inputChar is not supported
            throw new InvalidOperationException();
```

Reply:
Ok, now it's clear why there are no messages in exceptions in nano. Fixed.